rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_are_all_terminal_fan_configs_parallel.py

```diff
@@ -0,0 +1,46 @@
+from rct229.ruleset_functions.baseline_systems.baseline_hvac_sub_functions.are_all_terminal_fan_configs_parallel import (
+    are_all_terminal_fans_config_parallel,
+)
+
+TEST_RMD = {
+    "id": "test_rmd",
+    "buildings": [
+        {
+            "id": "building_1",
+            "building_segments": [
+                {
+                    "id": "building_segment_1",
+                    "zones": [
+                        {
+                            "id": "zone 1",
+                            "terminals": [
+                                {"id": "terminal_1", "fan_configuration": "PARALLEL"},
+                                {"id": "terminal_2"},
+                                {"id": "terminal_3", "fan_configuration": "SERIES"},
+                            ],
+                        },
+                    ],
+                }
+            ],
+        }
+    ],
+}
+
+TEST_RMD_FULL = {"id": "229_01", "ruleset_model_instances": [TEST_RMD]}
+
+
+def test__all_terminal_fans_parallel():
```

Comment:
I think the test function name should match up with the name of the function being tested.